.github/workflows/main.yml

```
@@ -13,72 +12,31 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-go@v5
-        with:
-          go-version-file: 'go.mod'
-
-      - name: Build
-        run: go build ./...
+    uses: omec-project/.github/.github/workflows/build.yml@main
+    with:
+      branch_name: ${{ github.ref }}
 
   staticcheck:
```

This was renamed in all other repos
```suggestion
  static-analysis:
```